src/js/imask.js

```diff
@@ -0,0 +1,4843 @@
+(function (global, factory) {
+	typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
+	typeof define === 'function' && define.amd ? define(['exports'], factory) :
+	(global = global || self, factory(global.IMask = {}));
+}(this, (function (exports) { 'use strict';
+
+	var commonjsGlobal = typeof globalThis !== 'undefined' ? globalThis : typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
```

Comment:
Codacy found an issue: ['globalThis' is not defined.](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5715280)